catalog.html

```diff
@@ -1,15 +1,383 @@
+                        <li><label><input type="checkbox" class="form-box-item">Travel & Tourism</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Health & Fitness</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">E-Commerce</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Education</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Entertainment</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Others</label></li>
+                    </ul>
+                </fieldset>
+            </form>
+            <hr>
+            <form class="filter-form">
+                <fieldset class="form-box">
+                    <legend class="form-box-title">Format Type</legend>
+                    <ul class="form-box-list">
+                        <li><label><input type="checkbox" class="form-box-item">All Formats</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Figma</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Sketch</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Adobe XD</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Adobe Photoshop</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Adobe Illustration</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Others</label></li>
+                    </ul>
+                </fieldset>
+            </form>
+            <hr>
+            <form class="filter-form">
+                <fieldset class="form-box">
+                    <legend class="form-box-title">Style</legend>
+                    <ul class="form-box-list">
+                        <li><label><input type="checkbox" class="form-box-item">All Styles</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Minimalist</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Modern</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Playful</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Dark Mode</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Photocentric</label></li>
+                        <li><label><input type="checkbox" class="form-box-item">Others</label></li>
+                    </ul>
+                </fieldset>
+            </form>
+            <hr>
+            <h2 class="price-heading">Price</h2>
+            <!-- I am not sure if this suppose to be heading since i didn't use heading in legend element? -->
```

Comment:
we will talk about form elements later 
but fieldset and legends is more preferable for using in forms